src/catkin_pkg/package.py

```diff
@@ -346,6 +346,14 @@ def __hash__(self):
     def __str__(self):
         return self.name
 
+    def __repr__(self):
+        kv = []
+        for slot in self.__slots__:
+            attr = getattr(self, slot, None)
+            if attr:
```

Comment:
Should this condition be changed to `attr is not None`? I would guess that it would be valuable to see when e.g. `evaluated_condition` has a value of `False`.

Reply:
You are right, that is valuable. I've modified the code and added a test.